src/api_server/src/parsed_request.rs

```diff
@@ -353,93 +353,98 @@ pub(crate) mod tests {
     #[test]
     fn test_error_into_response() {
         // Generic error.
-        let mut buf: [u8; 150] = [0; 150];
+        let mut buf: [u8; 145] = [0; 145];
```

Comment:
nit:

It'd be great to get rid of these manually-sized slices as well. It's time-consuming every time we need to change any parts of these error messages.

Reply:
I do agree. Addressed in 8d92db595c2777432e636c91b171d091b11b46bc